From 7bd9b4db16ed3c4d4b05d0ee955c08e115795f0b Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Thu, 27 Dec 2018 11:59:24 +0200
Subject: [PATCH 0832/1051] dts: a3700: remove USB3 node from a3700 B and C
 configurations

a3700 B and C configurations use Serdes 0 for SGMII and
USB3 that use same lane should be removed from device tree.

Change-Id: I32819b149bc222af10019649ffbc7c9bb100e19f
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/62149
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Liron Himi <lironh@marvell.com>
(cherry picked from commit e6539da9c5574c8088e7d93088ba8cb6c661c766)
Reviewed-on: http://vgitil04.il.marvell.com:8080/62387
Reviewed-on: https://sj1git1.cavium.com/1562
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db-B.dts | 4 +---
 arch/arm64/boot/dts/marvell/armada-3720-db-C.dts | 2 ++
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
index 427724d9d21f..dbe881209b04 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db-B.dts
@@ -24,6 +24,4 @@
 	};
 };
 
-&usb3 {
-	status = "disabled";
-};
+/delete-node/ &usb3;
diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db-C.dts b/arch/arm64/boot/dts/marvell/armada-3720-db-C.dts
index 937f4ea91185..552e9a6bb160 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db-C.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db-C.dts
@@ -15,3 +15,5 @@
 	phys = <&a3700_comphy0 1>;
 	managed = "in-band-status";
 };
+
+/delete-node/ &usb3;
-- 
2.17.1

